test/conftest.py

```diff
@@ -17,6 +18,11 @@
     sudo,
 )
 
+MACOS = False
+
+if platform.system() == "Darwin":
+    MACOS = True
```

Comment:
This should be importable from `.utils`

Reply:
Good cleanup, thanks. fc281b2fb8cad71253a3889420e9dba82739c954